app/jobs/audit_search_event_job.rb

```diff
@@ -1,11 +1,7 @@
 class AuditSearchEventJob < SpreadsheetWriterJob
```

Comment:
Shouldn't this inherit from `ApplicationJob` not `SpreadsheetWriterJob`?

Reply:
Nice catch. I don't think we need `SpreadsheetWriterJob` around at all now either

Reply:
Great catch again! Sorted.